src/peft/tuners/xlora/layer.py

```diff
@@ -0,0 +1,186 @@
+from typing import Any, Callable
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+from peft.tuners import lora
+
+from .config import XLoraConfig
+
+
+class XLoRALayer:
```

Comment:
Yeah, I think I like this new design of the `XLoraLayer`s much better, good work.

Reply:
Thanks!